source/includes/api/v3/_campaigns.md

```diff
@@ -629,3 +629,303 @@ skip | true | Data offset index. Useful for pagination.
+        "modifiedBy": "f7db3e84-e0f0-4b8b-94b2-2e5675ddddaa"
+    },
+    "customerId": "bd4b51e6-804c-497d-8f60-cf7b12c56e06",
+    "securityGroupId": "2e61dbbc-02f2-4084-b36e-c3d1d39ed495"
+}
+```
+### HTTP Request
+
+`GET https://api.beta.polisapp.com/v1/campaigns/{campaignId}/contacts/{contactId}/households/{householdId}`
+
+### Query Parameters
+
+Parameter | Required | Description
+--------- | -------- | -----------
+campaignId | true | The identifier of a campaign.
+contactId | true | The identifier of a contact.
+householdId | true | The identifier of a household.
+
+## Update a contact household campaign record
+
+<aside class="notice">
+You can only patch attributes inside <b>data</b> key.
+</aside>
+
+[RFC 6902 - JavaScript Object Notation (JSON) Patch](https://tools.ietf.org/html/rfc6902)
+
+> Example Request
+
+```http
+PATCH /v1/campaigns/c92162c6-f604-4caf-a964-d668d8bf67c8/contacts/cc20ab46-6233-491d-82ac-94c7ba6a8997/households/e50be735-cfe3-49db-8fca-e03f630d4a7e/status HTTP/1.1
+Host: api.beta.polisapp.com
+Authorization: Bearer {access_token}
+etag {etag value}
+{
+    "data": [
+        {
+        	 "op": "replace", "path": "/statusId", "value": "53a18d44-54f0-400e-b9cd-dd0bd32ae904" 
+        },
+        {
+        	"op": "replace", "path" : "/state", "value": "contacted-unavailable-not-home"
+        },
+        {
+        	"op": "replace", "path" : "/contacted", "value": true
+        }
+    ]
+}
+```
+
+> Example Response
+
+```json
+{
+    "id": "00001820-1329-490c-b07c-2071c29329de",
+    "data": {
+        "state": "contacted-unavailable-not-home",
+        "statusId": "53a18d44-54f0-400e-b9cd-dd0bd32ae904",
+        "contacted": true,
+        "campaignId": "c92162c6-f604-4caf-a964-d668d8bf67c8",
+        "successful": false,
+        "reservation": {
+            "reserved": false
+        },
+        "availability": {
+            "available": false,
+            "unavailableReason": null
+        },
+        "contactHouseholdId": "b66e4452-5fcd-4b79-bb1c-068ed7353192"
+    },
+    "meta": {
+        "created": "2019-09-05T20:54:33.580Z",
+        "modified": "2020-05-26T21:07:31.802Z",
+        "resource": "contact-household-campaigns",
+        "createdBy": "f7db3e84-e0f0-4b8b-94b2-2e5675ddddaa",
+        "isDeleted": false,
+        "modifiedBy": "ccce05b9-ddad-44e0-bc4e-7e9fe37edb4a",
+        "etag": "2e5-DCRtzEum5wZhZGFCH6Spy7M8nUE"
+    },
+    "customerId": "bd4b51e6-804c-497d-8f60-cf7b12c56e06",
+    "securityGroupId": "2e61dbbc-02f2-4084-b36e-c3d1d39ed495"
+}
+```
+
+### HTTP Request
+
+`PATCH https://api.beta.polisapp.com/v1/campaigns/{campaignId}/contacts/{contactId}/households/{householdId}`
+
+### Query Parameters
+
+Parameter | Required | Description
+--------- | -------- | -----------
+campaignId | true | The identifier of a campaign.
+contactId | true | The identifier of a contact.
+householdId | true | The identifier of a household.
+
+## Update or create new contact household campaign record
+
```

Comment:
Can we give a brief description here e.g. "Updates a contact-household-campaign record, if it exists. Otherwise, creates"

Reply:
please do clarify that it will update if the CHC exists - let's try to be super explicit